src/challenges/the_if_function.js

```diff
@@ -0,0 +1,17 @@
+function _if(bool, func1, func2) {
+  if (bool) {
+    func1()
+  } else {
+    func2()
+  }
+}
+
+function func1() {
+  console.log("function 1 was called")
```

Comment:
`func1` and `func2` shouldn't be a part of this file, because they are not important to the function. If you want to add them to the test file that would be okay.